reader_test.go

```diff
@@ -96,7 +96,40 @@ func TestDecodeMasterPlaylistWithAlternatives(t *testing.T) {
 			t.Fatal("should not be alternatives for this variant")
 		}
 	}
-	// fmt.Println(p.Encode().String())
+	//fmt.Println(p.Encode().String())
+}
+
+func TestDecodeMasterPlaylistWithClosedCaptionEqNone(t *testing.T) {
+	f, err := os.Open("sample-playlists/master-with-closed-captions-eq-none.m3u8")
+	if err != nil {
+		t.Fatal(err)
+	}
+	p := NewMasterPlaylist()
+	err = p.DecodeFrom(bufio.NewReader(f), false)
+	if err != nil {
+		t.Fatal(err)
+	}
+	// check parsed values
+	if p.ver != 4 {
+		t.Errorf("Version of parsed playlist = %d (must = 4)", p.ver)
+	}
+	if len(p.Variants) != 3 {
+		t.Fatal("not all variants in master playlist parsed")
+	}
+
+	for i, v := range p.Variants {
+		//spew.Dump(p.Variants)
+		if i == 0 && len(v.Alternatives) != 5 {
+			t.Fatalf("not all alternatives from #EXT-X-MEDIA parsed (has %d but should be 5", len(v.Alternatives))
+		}
+		if i > 0 && len(v.Alternatives) > 0 {
```

Comment:
Neither of these `if` blocks belong here, looks like they were from the previous test testing Alternatives, feel free to just have the one `if v.Captions ~!= ...`, that's all you need to test for.